gradle/spotless.gradle

```diff
@@ -0,0 +1,17 @@
+apply plugin: 'com.diffplug.spotless'
+spotless {
+
+    if (projectType.javaProjects.contains(project)){
+        java {
+            bumpThisNumberIfACustomStepChanges(1)
+            // currently we do not use spotless here, but our own bash solution - but spotless would be:
```

Comment:
What do you mean with `we do not use spotless here` do you mean for license files? 
What Bash file does "our solution"? Do you mean license headers?

Reply:
see my next comment